amp/WEB-INF/src/org/digijava/module/aim/action/CompareActivityVersions.java

```diff
@@ -101,7 +108,7 @@ public ActionForward compare(ActionMapping mapping, ActionForm form, HttpServlet
 
         vForm.setOutputCollection(new ArrayList<CompareOutput>());
         // Load the activities.
-        vForm.setActivityOne((AmpActivityVersion) session.load(AmpActivityVersion.class, vForm.getActivityOneId()));
+       /* vForm.setActivityOne((AmpActivityVersion) session.load(AmpActivityVersion.class, vForm.getActivityOneId()));
```

Comment:
as a general practice in order to be able to review code easier it would be good not to comment code, and just remove it. If you see in the pr page is complex to trace down what has changed in this pr